docs/contributtion/feature-flags.md

```diff
@@ -0,0 +1,67 @@
+# Feature Flags
+
+## Table of Contents
+
+- [Current feature flags](#current-feature-flags)
+- [Turning on feature flags in SASS](#turning-on-feature-flags-in-sass)
+
+The Spirit Design System codebase ships with a number of feature flags. These feature
+flags enable new behavior and styling, allowing you to opt-in to new (and
+sometimes breaking) changes while using the current version. A feature flag may
+be configured in React, SASS, or both.
+
+This section in the Storybook documents each feature flag that is available and
```

Comment:
I understand it is used in the storybook, but I think it should work on it own.
```suggestion
This section documents each feature flag that is available and
```

Reply:
Fixed.